packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -15,20 +15,25 @@ import {
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 import { gridClasses } from '../../gridClasses';
 
-interface GridExportFormat {
-  format: ExportTypes;
-  formatOptions?: GridCsvExportOptions;
-}
-
-type GridExportFormatOption = GridExportFormat;
+type GridExportDisplayOptions = {
+  /**
+   * If `true`, this export option will be removed from the GridToolbarExport menu.
+   * @default false
+   */
+  disableToolbarButton?: boolean;
+};
 
-type GridExportOption = GridExportFormatOption & {
+type GridExportOption = {
   label: React.ReactNode;
+  format: ExportTypes;
+  formatOptions?:
+    | (GridCsvExportOptions & GridExportDisplayOptions)
+    | (GridPrintExportOptions & GridExportDisplayOptions);
```

Comment:
It fits in one line.

```suggestion
  formatOptions?: (GridCsvExportOptions | GridPrintExportOptions) & GridExportDisplayOptions;
```